### PR TITLE
readme file second update

### PR DESCRIPTION
**Pull request by ralexr22**
practing pull requests.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,4 @@
 Line 1
+Line 2
+Line 3
```
Comment (alexstophel) [inline readme.md:3]: Wow! Awesome change!

File: readme.md [hunk 2]
```diff
@@ -1 +1,4 @@
 Line 1
+Line 2
```
Comment (alexstophel) [inline readme.md]: How about spelling this out?

**Comment by alexstophel** [comment]
👍

File: readme.md (hunk 1 above)
Comment (ralexr22) [inline readme.md:3]: Thansk!

**Pull request merged**